src/QMCWaveFunctions/lcao/LCAOrbitalBuilder.cpp

```diff
@@ -854,43 +895,42 @@ void LCAOrbitalBuilder::LoadFullCoefsFromH5(hdf_archive& hin, int setVal, PosTyp
 }
 
 /// Periodic Image Phase Factors computation to be determined
-void LCAOrbitalBuilder::EvalPeriodicImagePhaseFactors(PosType SuperTwist )
+void LCAOrbitalBuilder::EvalPeriodicImagePhaseFactors(PosType SuperTwist)
 {
-
 #if not defined(QMC_COMPLEX)
```

Comment:
This a prexisting #ifndef so leaving it hear doesn't violate the principle of not adding more things we don't want to the code, but a similar method as in SlaterDetBuilder would eliminate them here.

Reply:
I have fixed this one too since I have introduced the ifdef not long ago. Based on the previous conversation between @PDoakORNL and @ye-luo I used both templated and overload solutions for each ifdef.